applications-frontend/src/components/Applicant.vue

```diff
@@ -114,6 +117,15 @@ export default {
     };
   },
   computed: {
+    isReadOnly: function() {
+      return this.isReadOnlyForUser();
+    },
+    isDisabled: function() {
+      return !this.application.id || this.isReadOnlyForUser();
+    },
+    isAssignedDifferent: function() {
```

Comment:
I would prefer if you moved the method body into these methods

Reply:
Done